document-generator-interface/pom.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>com.companieshouse</groupId>
+    <artifactId>document-generator</artifactId>
+    <version>unversioned</version>
+  </parent>
+  <groupId>uk.gov.companieshouse</groupId>
+  <artifactId>document-generator-interface</artifactId>
+  <version>unversioned</version>
+  <name>document-generator-interface</name>
+  <url>http://maven.apache.org</url>
+  <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+  </properties>
+  <dependencies>
+    <dependency>
+      <groupId>junit</groupId>
+      <artifactId>junit</artifactId>
+      <version>3.8.1</version>
```

Comment:
This is an old version. Some of our repos are using JUnit 5. Version 3 doesn't support annotations (e.g. @Test)

Reply:
I agree here also, please use either 4 or 5. I would recommend 5 as it follows the version our `company-accounts` repos are using.